src/components/home/elements/TxInfoModal.vue

```diff
@@ -130,7 +130,7 @@
         </div>
       </div>
       <div class="tx-address"
-           v-if="((txType === 'Sent' || txType === 'Received') && tokenId)">
+           v-if="((txType === 'Sent' || txType === 'Received' || txType === 'Deposit' || txType === 'Withdraw') && tokenId)">
```

Comment:
The condition of if statement can simplify to `['Sent', 'Received', 'Deposit', 'Withdraw'].includes(txType) && tokenId`